### ISSUE TITLE
current build incorrectly installing aicsimageio version and dependencies

### ISSUE THREAD

**Issue by TimMonko** [haesleinhuepf/devbio-napari#14]
Hi @haesleinhuepf ,
Back again. CZIreader from AICSImageIO does not work in current conda build. 
 `conda create -name new-env -c conda-forge devbio-napari`. I have found the reason is due to 1) lacking install of xmlschema module and 2) some other weirdness... older forges result in aicsimageio 4.7.0 and aicspylibczi 3.0.5, consistent with the current versions installed by conda forge. Installing the missing xmlschema module does not fix the issue, so not sure what tangles are back there. 

Solution: `conda install aicsimageio` solves this by installing aicsimageio 4.9.1, aicspylibczi 3.0.5 and additionally installs xmlschema 1.4.1. Alternatively I had success with `conda create --name new-env -c conda-forge aicsimageio devbio-napari`

Is the way to solve this just by putting `aicsimageio>=4.9.1` in the requirements.txt?

**Comment by haesleinhuepf** [comment]
Hi Tim @TimMonko ,

we cannot add `aicsimageio` due to licensing incompatibly. Sad I know.

See #5 

> CZIreader from AICSImageIO does not work in current conda build.
conda create -name new-env -c conda-forge devbio-napari. 

Can you describe the problem a bit in detail? And consider telling the aicsimageio folks about it. Your description sound like their installation is broken ...

**Comment by TimMonko** [comment]
@haesleinhuepf thanks for the reply. 

To describe the problem in more detail...
I think it was my doing ;)
At some point I must have accidentally pip installed aicspylibczi in my base python directory. As such, `conda list` exposed the conda installed packages... but masked them if there was a package in the base python `envs` folder. I discovered this by comparing `conda list` and `pip list`. 

Now doing it correctly (and reinstalling everything, this time with miniconda) `conda list` correctly shows packages install with pip _and_ properly displays the source (pip, conda, etc). Previously with the packages in the base directory, `conda list` displayed the source as conda-forge, but would load the older version masked package. 

It  finally makes sense why building devbio-napari on different machines caused many different results -- depended if I bricked the base directory back when I was learning python or not. 

Cheers :)
